importer/README.md

```diff
@@ -174,6 +174,7 @@ The coverage report `coverage.html` will be at the working directory
 - Adding a value to the Location column will create a separate List resource (or update) that links the inventory to the provided location resource
 - A separate List resource with references to all the Group and List resources generated is also created
 - You can pass in a `list_resource_id` to be used as the identifier for the (reference) List resource, or you can leave it empty and a random uuid will be generated
+- You can pass in the `link_list_resources` as `false` to avoid linking adding the Group and Linkage List resource being linked to the `list_resource_id`.  This is mostly suitable when syncing with `Location` and `RelatedEntityLocation` sync strategy
```

Comment:
Is it possible to re-word the first sentence? It's a bit confusing.

Reply:
Updated!!